haystack/management/commands/update_index.py

```diff
@@ -74,21 +76,24 @@ def worker(bits):
 
 
 def do_update(backend, index, qs, start, end, total, verbosity=1):
+    start_time = now()
     # Get a clone of the QuerySet so that the cache doesn't bloat up
     # in memory. Useful when reindexing large amounts of data.
     small_cache_qs = qs.all()
     current_qs = small_cache_qs[start:end]
 
     index.pre_process_data(current_qs)
 
+    # FIXME: Get the right backend.
+    backend.update(index, current_qs)
```

Comment:
I moved the line with the comment.  I have no idea what they think is broken with this.